src/users/views.py

```diff
@@ -116,7 +116,8 @@ def post(self, request):
         mail.template_id = USER_VERIFICATION_TEMPLATE
         sg = sendgrid.SendGridAPIClient(apikey=settings.SENDGRID_API_KEY)
         response = sg.client.mail.send.post(request_body=mail.get())
-
+        if response.status_code != 202:
+            raise ValidationError({'status_code': response.status_code})
```

Comment:
can you make it like 
```python
if not (200 <= response.status_code < 200):
    raise ValidationError({'status_code': response.status_code})
```

I'm worried about other 200 status codes.  200 should always be success.

Reply:
Also, probably out of scope of this PR, but it would be nice if there was an easy helper function to send an email given all of these parameters.  Like, we're always going to have it be `'text/html'`, we're always going to use a template, so forth and so on